aws/validators.go

```diff
@@ -654,6 +655,55 @@ func validateAwsAccountId(v interface{}, k string) (ws []string, errors []error)
 	return
 }
 
+func validateKinesisAnalyticsRuntime(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Can simplify as

```go
var validateKinesisAnalyticsRuntime = validation.StringInSlice([]string{
	kinesisanalyticsv2.RuntimeEnvironmentSql10,
	kinesisanalyticsv2.RuntimeEnvironmentFlink16,
	kinesisanalyticsv2.RuntimeEnvironmentFlink18,
}, false)
```

and the same for the other multi-use validators added here - We can then drop the additional unit tests.